src/ResourceManager/Sql/Commands.Sql/Auditing/Cmdlet/GetAzureSqlDatabaseAuditingPolicy.cs

```diff
@@ -21,6 +22,7 @@ namespace Microsoft.Azure.Commands.Sql.Auditing.Cmdlet
     /// Returns the auditing policy of a specific database.
     /// </summary>
     [Cmdlet(VerbsCommon.Get, "AzureRmSqlDatabaseAuditingPolicy", SupportsShouldProcess = true), OutputType(typeof (AuditingPolicyModel))]
+    [Obsolete]
```

Comment:
Yes, please add in a comment that specifies the cmdlet name, that it is osbsolete, and the new cmdlet name, like:

```c#
[Obsolete("Get-AzureSqlDatabaseAuditingPolicy is obsolete.  It will be removed in a future release. Please use the Get-AzureSqlDatabaseAuditing cmdlet instead. ", false)]
```